### PR TITLE
Add allow/block list support to org settings

### PR DESCRIPTION
**Pull request by ryanjduffy**
Blocked by https://github.com/RecordReplay/backend/pull/5639

Adds input fields to the org settings pane to set allow / block lists

https://app.replay.io/recording/add-allowblock-list-to-org-settings--598e9fa4-af0a-4cf5-988e-8822d94b8ff7

### CODE REVIEW

File: src/ui/components/shared/WorkspaceSettingsModal/OrganizationSettings.tsx [hunk 1]
```diff
@@ -1,13 +1,90 @@
...
+    "//" +
+    // domain - adapted from https://stackoverflow.com/questions/3117218/matching-url-with-wildcards
+    // full supported list https://www.icann.org/en/system/files/files/idna-protocol-2003-2008.txt
+    "([*\\w\\d\\-\\u0100-\\uffff.%]*)" +
+    // port
+    "(?::([*0-9]+))?",
+  "u"
+);
+
+function sanitizeUrlList(input: string) {
```
Comment (jaril) [inline src/ui/components/shared/WorkspaceSettingsModal/OrganizationSettings.tsx:22]: Is it worth adding tests here?

**Review by jaril — APPROVED** [review]
LGTM! Left a comment re:unit tests but not blocking.

**Comment by jaril** [comment]
@ryanjduffy circling back to this — anything stopping this from being merged? minus the test comment but just making sure that there's no other blocker :)

**Comment by ryanjduffy** [comment]
Just time :)

**Pull request merged**